docs/content/preview/admin/yb-ts-cli.md

```diff
@@ -280,12 +280,19 @@ The duration, in milliseconds (ms), before the RPC request times out.
 
 Default: `60000` (1000 ms = 1 sec)
 
+##### --certs_dir_name
+
+To connect to a cluster with TLS enabled, you must include the `-certs_dir_name` flag with the directory location where the root certificate is located.
+
+Default: `""`
```

Comment:
what does this mean as a default? empty? there is no default?

Reply:
Look the same option in yb-admin https://docs.yugabyte.com/preview/admin/yb-admin/. It says the same thing there.

Reply:
Okay - so it's some form of empty